docs/user-guide/cli-configuringcli-ev.md

```diff
@@ -23,9 +23,9 @@ You can set the location on your computer where Zowe CLI creates the `.zowe` dir
 
 ## Setting a shared plug-in directory
 
-You can set the location of a shared directory to manage plug-ins for multiple users.
+You can set the location of a shared directory to manage Zowe CLI plug-ins for multiple users.
```

Comment:
How about instead of “You can set the location …,” you say “Set the location …”?